cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2219,6 +2219,10 @@ public static final class Event {
     public static final String EVENTS_WRITER_EXTENSIONS_DIR = "events.writer.extensions.dir";
 
     public static final String EVENTS_WRITER_EXTENSIONS_ENABLED_LIST = "events.writer.extensions.enabled.list";
+
+    public static final String EVENTS_READER_PREFIX = "event.reader";
+
+    public static final String EVENTS_READER_ACK_BUFFER = "events.reader.buffer";
```

Comment:
You need to set this in cdap-default.xml.

Reply:
Both constants?